itou/prescribers/management/commands/sync_ft_organizations.py

```diff
@@ -0,0 +1,150 @@
+            if wet_run:
+                try:
+                    with transaction.atomic():
+                        organization.save(update_fields={"code_safir_pole_emploi"})
+                except IntegrityError:
+                    self.stdout.write(f"ERR: safir={organization.code_safir_pole_emploi} already used")
+
+    def update_information(self, data, *, wet_run=False):
+        qs = PrescriberOrganization.objects.filter(kind=PrescriberOrganizationKind.PE).exclude(
+            Q(code_safir_pole_emploi="") | Q(code_safir_pole_emploi=None)
+        )
+        mapping = [
+            ("siret", "siret"),
+            (name_from_api_data, "name"),
+            (phone_from_api_data, "phone"),
+            (email_from_api_data, "email"),
+            (address_from_api_data, "address_line_1"),
+            (address_extra_from_api_data, "address_line_2"),
+            (post_code_from_api_data, "post_code"),
+            (coordinates_from_api_data, "coords"),
+            (insee_city_from_api_data, "insee_city"),
+        ]
+        for item in yield_sync_diff(
+            [datum for datum in data if datum.get("siret")], "codeSafir", qs, "code_safir_pole_emploi", mapping
+        ):
+            if item.kind == DiffItemKind.DELETION:  # Ignore DELETION completely
+                continue
+            self.stdout.write(item.label)
+            if item.kind == DiffItemKind.SUMMARY:  # Nothing more to do than display a message for SUMMARY
+                continue
+
+            safir = item.raw["codeSafir"]
+            siret = item.raw["siret"]
+            # Some agencies (with different location) share the same siret, this trigger the (siret, kind) unique
+            # constraint, as we don't really care about siret for France Travail organization we circumvent that by
+            # clearing the siret for the current one if it doesn't already exist or if is not the first one we see.
+            if (
+                siret
+                and PrescriberOrganization.objects.filter(kind=PrescriberOrganizationKind.PE, siret=siret)
+                .exclude(code_safir_pole_emploi=safir)
+                .exists()
+            ):
+                siret = None
+            if item.kind == DiffItemKind.ADDITION:
+                obj = PrescriberOrganization(
+                    kind=PrescriberOrganizationKind.PE,
+                    code_safir_pole_emploi=safir,
+                    is_authorized=True,
+                    authorization_status=PrescriberAuthorizationStatus.VALIDATED,
+                )
+                fill_organization_from_api_data(obj, siret, item.raw)
+                if wet_run:
+                    obj.save()
+            elif item.kind == DiffItemKind.EDITION:
+                fill_organization_from_api_data(item.db_obj, siret, item.raw)
+                if wet_run:
+                    item.db_obj.save()
+
+    @transaction.atomic
+    def handle(self, *, action, wet_run=False, **options):
```

Comment:
Le défaut est fourni par `add_argument`.
```suggestion
    def handle(self, *, action, wet_run, **options):
```

Reply:
Ici pourquoi pas, mais en général j'applique la même logique de "si c'est une option alors je le rend optionnel" sinon ça devient très vite relou de devoir tout passer à chaque fois que tu appelles la fonction dans les tests.